### DISCUSSION TITLE
Bandpass filter for MP3 streams?

### DISCUSSION THREAD

**Discussion by ivellios1988** [savonet/liquidsoap#1750] [Q&A] [answered]
In my radio I have a stream which broadcasts in MP3 with 128kbps. I'd like to reduce the amount of audible distortion in stereo signal, by cutting the audio bandwidth to, say, 14kHz or even 13kHz. Is there any way to achieve this in Liquidsoap?

**Comment by toots** [comment, marked as answer]
Hi! Have you considered using a ladspa high-pass filter for it? There's a comparison of some of them here it seems: https://forum.renoise.com/t/high-pass-filter-comparison-renoise-native-and-ladspa-with-pictures/21942

**Comment by smimram** [comment]
Also Liquidsoap features a number of filters. I'd advise using the butterworth filters such as `filter.iir.butterworth.bandpass` (or `.high` or `.low` depending on whether you want a high-pass or a low-pass filter).
